canary/ClientApp/src/components/tests/Connectathon.js

```diff
@@ -159,14 +160,16 @@ export class Connectathon extends Component {
                 <Header as="h2" dividing id="step-1">
                   <Icon name="flag" />
                   <Header.Content>
-                    Step 1: Select State
+                    Step 1: Set Certificate Number and Select Jurisdiction
                     <Header.Subheader>
-                      Select the state which you are generating a message from.
+                      Specify the certificate number to be used in the record and select the jurisdiction which you are generating a record from.
                     </Header.Subheader>
                   </Header.Content>
                 </Header>
-                <div className="p-b-15" />
-                <Dropdown placeholder='Select State' search selection fluid onChange={this.fetchTest} options={stateOptions} />
+                <h4>Certificate Number</h4>
+                <Input value={this.state.certificateNumber} placeholder='Enter Certificate Number' fluid onChange={this.updateCertificateNumber} />
```

Comment:
Might be beneficial to add `type="number"` to this, in its current state if you add `$` or `\` it throws a 404 error, so eliminating all possible characters except numbers may be good.

Reply:
Done!